projects/v2/src/app/modules/root/root.component.ts

```diff
@@ -190,7 +191,8 @@ export class RootComponent implements OnInit, OnDestroy{
 
     this.bs$.subscribe(value => {
       if (value) {
-        const bsd = store.selectSnapshot(TreeState.getBottomSheetData);
+        var bsd = store.selectSnapshot(TreeState.getBottomSheetData);
+        bsd['obser']= this.bottomSheetInfo$
```

Comment:
I see what is happening here. Refactor the code such that the title is set in the `bottomSheetData` after the API call. Remove the bottomSheetData from the `TreeState`. Keep everything in the sheet state.

This way you ca just pass `data: this.bottomSheetInfo$` instead of writing these extra lines of code